### DISCUSSION TITLE
How should multiple groups be implemented

### DISCUSSION THREAD

**Discussion by WooHoo-Wu** [tailwindlabs/tailwindcss#9679] [Help] [answered]
I have a scenario where I use multiple groups, and I don't want the children to be affected when the parent hovers, as in the following code：

```
<div class="w-10 h-10 group">
     <div class="w-8 h-8 group-hover:bg-red">
             <div class="w-6 h-6 group">
                    <div class="w-4 h-4 group-hover:bg-green"></div>
             </div>
     </div>
</div>
```

There are 4 levels here
I hope：
- When hovering the first layer, the second layer becomes red, and the fourth layer does not change
- When hovering the third layer, the color of the fourth layer changes

But：
The fourth layer changed color when I hovered the first one, which I didn't want

Do you have any solutions?

**Comment by wongjn** [comment, marked as answer]
You could look at [adding names to the groups](https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups) to differentiate them.
